packages/moonstone/ContextualPopupDecorator/ContextualPopupDecorator.js

```diff
@@ -555,6 +557,17 @@ const Decorator = hoc(defaultConfig, (config, Wrapped) => {
 			if (!Spotlight.focus(activator)) {
 				Spotlight.focus();
 			}
+
+			this.setState({
+				activatorAriaHidden: false
+			});
+
+			setTimeout(() => {
+				if (Spotlight.getCurrent() === activator) {
+					Spotlight.getCurrent().blur();
+					Spotlight.focus(activator);
+				}
+			}, 800);
```

Comment:
Could you please leave the reason to choose the time of 800ms in PR description?